iosApp/iosApp/Pages/NearbyTransit/NearbyTransitView.swift

```diff
@@ -205,6 +210,7 @@ struct NearbyTransitView: View {
             predictionsRepository.connect(stopIds: Array(stopIds)) { outcome in
                 DispatchQueue.main.async {
                     if let data = outcome.data {
+                        lastPredictions = Date.now.toKotlinInstant()
```

Comment:
Huh. I'd been thinking of the repositories as being stateless, but I guess the channel-based repositories already aren't, and nothing ever needs to rerender solely because its last updated time has changed, so that'd probably work.

Reply:
Yeah fair, maybe they're technically use cases. It's probably not worth renaming them all to use cases if you don't want to make the repos blatantly stateful though. I'm fine if you want to stick with this, but I'd prefer if the views didn't need to handle it.